pkg/apis/kops/v1alpha2/cluster.go

```diff
@@ -109,9 +109,11 @@ type ClusterSpec struct {
 	NonMasqueradeCIDR string `json:"nonMasqueradeCIDR,omitempty"`
 
 	// SSHAccess determines the permitted access to SSH
-	// Currently only a single CIDR is supported (though a richer grammar could be added in future)
```

Comment:
It this changing in this PR?

Reply:
No, but I spotted that this comment was wrong and I figured it wasn't _entirely_ unrelated ;-)